### ISSUE TITLE
bun repl crashes for me

### ISSUE THREAD

**Issue by guilt** [oven-sh/bun#12078]
### How can we reproduce the crash?

I ran `bun repl` after an upgrade and it crashed for me.

Tested on `Windows 11 23H2 22631.3737`

### Relevant log output

```shell
============================================================
Bun v1.1.16 (bf7b327f) Windows x64
Args: "bun" "bun-repl"
Elapsed: 8ms | User: 0ms | Sys: 0ms
RSS: 17.58MB | Peak: 17.58MB | Commit: 60.96MB | Faults: 4450

panic(main thread): Segmentation fault at address 0x7FF777EEFEB8
oh no: Bun has crashed. This indicates a bug in Bun, not your code.

To send a redacted crash report to Bun's team,
please file a GitHub issue using the link below:

 https://bun.report/1.1.16/wB1bf7b327AACiB\VCRUNTIME140.dllgxpEgjnnEoxwnFqiqiGqlfi8Io/xh+CCa\KERNEL32.DLL63yECU\ntdll.dllwy3WA2u//Bwr/793D
```

### Stack Trace (bun.report)

Bun v1.1.16 ([`bf7b327`](<https://github.com/oven-sh/bun/tree/bf7b327f68568104a5e40c1944b7b255b02f0745>)) on windows x86_64 [BunxCommand]

Segmentation fault at address 0x7FF777EEFEB8

- ??? at `0x11310` in VCRUNTIME140.dll
- [`install.zig:9601`](<https://github.com/oven-sh/bun/blob/bf7b327f68568104a5e40c1944b7b255b02f0745/src/install/install.zig#L9601>): `parse`
- [`bunx_command.zig:280`](<https://github.com/oven-sh/bun/blob/bf7b327f68568104a5e40c1944b7b255b02f0745/src/cli/bunx_command.zig#L280>): `exec`
- [`cli.zig:1527`](<https://github.com/oven-sh/bun/blob/bf7b327f68568104a5e40c1944b7b255b02f0745/src/cli.zig#L1527>): `start`
- [`cli.zig:62`](<https://github.com/oven-sh/bun/blob/bf7b327f68568104a5e40c1944b7b255b02f0745/src/cli.zig#L62>): `start`
- [`main.zig:51`](<https://github.com/oven-sh/bun/blob/bf7b327f68568104a5e40c1944b7b255b02f0745/src/main.zig#L51>): `main`
- [`exe_common.inl:78`](<https://github.com/oven-sh/bun/blob/bf7b327f68568104a5e40c1944b7b255b02f0745/src/src/startup/exe_common.inl#L78>): `invoke_main`
- ??? at `0x1257d` in KERNEL32.DLL
- ??? at `0x5af28` in ntdll.dll

**Comment by Jarred-Sumner** [comment]
Duplicate of #12070

**Comment by Jarred-Sumner** [comment]
This was fixed in #12071 

To upgrade, please run `bun upgrade --canary`
